cardinal/shard/adapter.go

```diff
@@ -24,21 +25,25 @@ type Adapter interface {
 
 // WriteAdapter provides the functionality to send transactions to the EVM base shard.
 type WriteAdapter interface {
+	// Submit submits a transaction to the EVM base shard's game tx sequencer, where the tx data will be sequenced and
+	// stored on chain.
 	Submit(ctx context.Context, p *sign.SignedPayload, txID, epoch uint64) error
 }
 
 // ReadAdapter provides the functionality to read transactions from the EVM base shard.
 type ReadAdapter interface {
+	// QueryTransactions queries transactions stored on-chain. This is primarily used to rebuild state during world
+	// recovery.
 	QueryTransactions(
 		context.Context,
 		*shardtypes.QueryTransactionsRequest) (*shardtypes.QueryTransactionsResponse, error)
 }
 
 type AdapterConfig struct {
-	// ShardReceiverAddr is the address to communicate with the secure shard submission channel.
-	ShardReceiverAddr string `json:"shard_receiver_addr,omitempty"`
+	// ShardSequencerAddr is the address of the base shard's game tx sequencer.
+	ShardSequencerAddr string `json:"shard_receiver_addr,omitempty"`
```

Comment:
ShardSequencerAddr does not match shard_receiver_addr

Reply:
got it - thanks for catching that